homework-2/__tests__/middlewares/groupMiddlewares.spec.ts

```diff
@@ -0,0 +1,297 @@
+import { Request, Response, NextFunction } from "express";
+import {
+    addGroupHandler,
+    addUserToGroupHandler,
+    getAllGroupsHandler,
+    getGroupByIdHandler,
+    removeGroupHandler,
+    updateGroupHandler
+} from "../../middewares/groupMiddlewares";
+import GroupService from './../../services/groupService';
+import { db } from "../../models";
+import { IdParam, GroupReqAttrs, Permission, GroupAttrs } from "../../types";
+
+describe('Group middlewares', () => {
+    describe('getAllGroupsHandler', () => {
+        let mockRequest: Partial<Request>;
+        let mockResponse: Partial<Response>;
+        const nextFunction: NextFunction = jest.fn();
+
+        beforeEach(() => {
+            mockRequest = {};
+            mockResponse = {
+                send: jest.fn()
```

Comment:
I would check mock send function is called or not

Reply:
I do it at line 63